### PR TITLE
refactor: refactor api position

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### Things to Talk About
moved API call position.

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/components/SecurityLSB.vue [hunk 1]
```diff
@@ -71,10 +76,46 @@ const state = reactive({
...
+        securityPageStore.setSelectedCloudServiceType(pageParams.group, pageParams.name);
+    }
+});
 watch(() => state.favoriteOptions, (favoriteOptions) => {
     gnbStore.setFavoriteItemId(favoriteOptions);
 }, { immediate: true });
+
+onBeforeMount(() => {
+    initData();
+});
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/components/SecurityLSB.vue:118]: I think this may need Scope Guard

**Review by piggggggggy — APPROVED** [review]
LGTM! I left a comment

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**